lib/node_modules/@stdlib/napi/argv-complex64/manifest.json

```diff
@@ -0,0 +1,42 @@
+{
+	"options": {},
+	"fields": [
+		{
+			"field": "src",
+			"resolve": true,
+			"relative": true
+		},
+		{
+			"field": "include",
+			"resolve": true,
+			"relative": true
+		},
+		{
+			"field": "libraries",
+			"resolve": false,
+			"relative": false
+		},
+		{
+			"field": "libpath",
+			"resolve": true,
+			"relative": false
+		}
+	],
+	"confs": [
+		{
+			"src": [
+       			"./src/main.c"
```

Comment:
You have mixed TAB and space indentation. This file should be 2-space indentation.